### PR TITLE
WIP – Gigabit Build

### LINKED ISSUE #2228: Build Gigabit content merge (/gigabit)
**Staged**: https://learning-mozilla-org-s-pr-2298.herokuapp.com/en-US/gigabit

Redpen: https://redpen.io/p/ku41f3c8ea603732e1
Related: https://github.com/mozilla/learning.mozilla.org/issues/2058
Sketch: https://drive.google.com/a/mozillafoundation.org/file/d/0B4Q8pzCpDS_eRmtDV1dmcTgxb1k/view

### PR DESCRIPTION
**Pull request by gvn**
This adds the new /gigabit section.

**WIP! DON'T MERGE**

Closes #2228

### CODE REVIEW

**Comment by gvn** [comment]
@flukeout This is ready for code review!

File: less/pages/gigabit.less [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+    &.mui-active {
+      background-color: @white;
+    }
+  }
+}
+
+.gigabit-contact {
+  h4 {
+    font-size: 26px;
```
Comment (flukeout) [inline less/pages/gigabit.less]: Should we do ems for font-size?

File: less/pages/gigabit.less [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    font-size: 18px;
+  }
+
+  .lead-org, .lead-org a, .summary {
+    line-height: 2em;
+    font-size: 12px;
+  }
+}
+
+.tag {
```
Comment (flukeout) [inline less/pages/gigabit.less]: Put this into the .tags block below?

File: less/pages/gigabit.less [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+    margin-right: 18px;
+    width: 36px;
+  }
+}
+
+.matrix {
+  width: 100%;
+
+  thead {
+    background: rgba(227, 131, 0, .1);
```
Comment (flukeout) [inline less/pages/gigabit.less]: Second time this color appears, should we make it a variable?

File: pages/gigabit/Terms.jsx [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+          <h2>Mozilla Gigabit Community Fund</h2>
+          <p className="m-b-3">Terms and Conditions</p>
+        </div>
+
+        <article>
+          <section className="terms-section">
+            <div>
+            <h4 className="section-header">Overview:</h4>
+            <p>
+            Mozilla Foundation (“Mozilla” or “Sponsor”) is a global non-profit organization with a mission to promote openness, innovation and opportunity on the Web. With support from the United States National Science Foundation (“NSF”), Mozilla presents this Gigabit Community Fund (the “Fund”) to help foster and support the development of applications and associated curricula running on next-generation networks that could improve people’s everyday lives through creating new education and workforce development opportunities. Proposals submitted to the Gigabit Community Fund may be based on projects funded in the separate Mozilla Ignite Challenge conducted prior to this Gigabit Community Fund, building off existing ideas and funding, or may be based entirely on new ideas (“Submissions”).
```
Comment (flukeout) [inline pages/gigabit/Terms.jsx]: Indent text in these paragraphs?

File: pages/gigabit/Terms.jsx [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+              <li>Does the project focus on education or workforce development; and</li>
+              <li>Does the project leverage next generation networking technologies like ultra high speed networking and software-defined networking.</li>
+            </ol>
+            <p>
+            Those Submissions meeting these criteria will be eligible for further review by a panel of local evaluators and representatives from Mozilla and NSF, and may include additional category-expert reviewers based on the following additional criteria:
+            </p>
+            <h5>Intellectual Merit</h5>
+            <ol>
+              <li>
+              <strong>Project Feasibility:</strong> Can the outlined project be completed in the pilot period given the project timeline and the background of the involved team members? Does the project team possess the necessary experience to work with the target audience? Are the project’s goals and outcomes attainable and measurable? Will the project team be able to achieve these results? Does the project location have gigabit internet?
```
Comment (flukeout) [inline pages/gigabit/Terms.jsx]: Indent text in these list items?

File: pages/gigabit/portfolio.json [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+    "Outcomes": "Viditor was accepted into UTC’s College of Computer Science & Engineering Senior Capstone program, where a team of students has joined the app founders in adding additional cloud functionality. And while still a work in progress, Viditor demoed at the Creative Discovery Museum during Startup Week, with Andrew live-programming additional camera filters for an enthusiastic audience, as well as at the Fiber to the Home Council conference in Ft. Lauderdale, FL this June. The project has since been piloted in an art class at the Baylor School and with beta-testers at the Chattanooga Public Library.",
+    "Gigabit Advantage": "Gigabit speeds allow for media-rich collaboration in real time, while low latency allows for simultaneous editing across multiple users even if in different places in CHA and beyond.",
+    "Lead Organization": "GeonCode",
+    "Other Partner Organizations": "University of Tennessee at Chattanooga, Baylor School, Chattanooga Public Library",
+    "Photo": "",
```
Comment (flukeout) [inline pages/gigabit/portfolio.json]: Why not null like used for Link?

File: pages/gigabit/portfolio.json [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+    "Project": "Gigabots",
+    "Project Summary": "The Gigabots bring connected devices to classrooms using existing educational robotics platforms. A Gigabot is a combination of hardware, software and a cloud platform that connects Gigabots to each other from any location in real-time.",
+    "Outcomes": "Students participating in the pilot learned basic JavaScript, how to program the Gigabots remotely, and how to implement the touch sensor into programming robot behaviors. This project was demoed at MakerFaire KC, at SummerQuest 2014 in the Bronx, as well as at the 2015 and 2016 US Ignite Summits.",
+    "Gigabit Advantage": "Gigabots expand traditional robotics-league curriculum with IoT and programming.  With sensor-based interactions, high speed and low latency are critical to the user experience. “Gigabots Live” offers a low cost, low latency video streaming device suitable for use with internet connected robotics devices.",
+    "Lead Organization": "Big Bang",
+    "Other Partner Organizations": "Staley High School",
+    "Photo": "/img/pages/gigabit/logos/14715602331_fe3448fd97_o.jpg",
+    "Link": "https://getbigbang.com/",
+    "Tech Tags": "Robotics, IoT",
```
Comment (flukeout) [inline pages/gigabit/portfolio.json]: Why not array of tags?

Comment (gvn) [inline pages/gigabit/portfolio.json]: I guess we could...currently this isn't used by anything actually. I figured we'd array-ify it once we make it into a real service.

File: pages/gigabit/portfolio.json (hunk 6 above)
Comment (gvn) [inline pages/gigabit/portfolio.json]: No reason...I could make it `null` for consistency

File: less/pages/gigabit.less (hunk 1 above)
Comment (gvn) [inline less/pages/gigabit.less]: It's kind of inconsistent in this codebase. If anything it'd be `rems` or `px`.

**Comment by flukeout** [comment]
Hey @gvn had a look over and left some minor notes on markup and CSS. As a certified React novice, this works look solid!

**Pull request merged**